Common/src/main/java/com/almostreliable/unified/AlmostUnified.java

```diff
@@ -13,6 +15,16 @@ public final class AlmostUnified {
     @Nullable private static AlmostUnifiedRuntime RUNTIME;
     @Nullable private static TagManager tagManager;
 
+    private static StartupConfig startupConfig;
+
+    static void initStartupConfig() {
+        startupConfig = Config.load(StartupConfig.NAME, new StartupConfig.Serializer());
+    }
+
+    public static StartupConfig getStartupConfig() {
+        return startupConfig;
```

Comment:
I'd move the `initStartupConfig` functionality into this accessor in case it's still `null`.